src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Functions for ordering and styling clinical attributes.
+ */
+/**
+ * Order patient clinical attributes by an object mapping.
+ * If the key does not exist in the map, then it shall be ordered last.
+ * @param {string} key
+ */
+export const getOrder = key => {
+    // Keys with lower numbers appear earlier
+    const map = {
+        SEX: 1,
+        GENDER: 1,
+        AGE: 2,
+        CANCER_TYPE: 3,
+        CANCER_TYPE_DETAILED: 4,
+        KNOWN_MOLECULAR_CLASSIFIER: 5,
+        HISTOLOGY: 6,
+        OS_STATUS: 7,
+        OS_MONTHS: 8,
+        DFS_STATUS: 9,
+        DFS_MONTHS: 10,
+    };
+
+    return key in map ? map[key] : Number.MAX_VALUE;
+};
+
+/**
+ * Compare two keys a and b by their mapped ordering in ascending order.
+ * For instance, GENDER is displayed before OS_MONTHS.
+ * @param {string} a
+ * @param {string} b
+ */
+export const compare = (a, b) => {
+    return getOrder(a) - getOrder(b);
+};
+
+/**
+ * Builds a string to insert in an HTML element.
+ * @param {string} value
+ * @param {string} key
+ * @param {object} config
+ */
+export const stringBuilder = (value, key, config) => {
+    var builder = [];
+
+    // Show comma before primary and not secondary attributes
+    const status = key in config ? config[key].primary : true;
+    const formatter =
+        key in config && 'formatter' in config[key]
+            ? config[key].formatter
+            : t => {
+                  return `${t}`;
+              };
+    const prefix = status !== true ? '' : ', ';
+
+    builder.push(prefix);
+    builder.push(formatter(value));
+    return builder.join('');
+};
+
+/**
+ *  Gets a color for the given key, value, and config. 
+    If there a specified color for the key and value, return it.
+    If there is no such value, then get the color only specified for the key. 
+    Otherwise, return a default color. 
+ * @param {string} value 
+ * @param {string} key 
+ * @param {object} config 
+ */
+export const colorGetter = (value, key, config) => {
+    // Style based on attr-id only and not attr-value
+    const color = 'black';
+    const NO_VALUE = '_';
+    const existsKey = key in config && 'values' in config[key];
+
+    return existsKey && value in config[key].values
+        ? config[key].values[value]
+        : existsKey && NO_VALUE in config[key].values
+        ? config[key].values[NO_VALUE]
+        : color;
+};
+
+// Colors taken from clinicalAttributes.scss
+export const colors = {
```

Comment:
this map isn't necessary.  just put this in the config structure and use strings (e.g. 'black')